ouroboros-network/src/Ouroboros/Network/NodeToClient/Version.hs

```diff
@@ -39,6 +39,9 @@ data NodeToClientVersion
     -- ^ enabled @CardanoNodeToClientVersion6@, adding a query
     | NodeToClientV_8
     -- ^ 'LocalStateQuery' protocol codec change, allows to acquire tip point.
+    | NodeToClientV_9
+    -- ^ 'LocalStateQuery' protocol codec change, queries are now
```

Comment:
Depending on where our merges fall between node releases, we may be able to add the new top-level queries to `NodeToClientV_9` as well.

Reply:
See also my comment [here](https://github.com/input-output-hk/ouroboros-network/pull/2975#discussion_r623812670).

Reply:
> IIUC, nothing needs to change here yet.

I think I came to that same conclusion, eg here https://github.com/input-output-hk/ouroboros-network/pull/3106#discussion_r629762791